public/hooks/use_chat_actions.tsx

```diff
@@ -106,6 +106,15 @@ export const useChatActions = (): AssistantActions => {
     }
   };
 
+  const resetChat = () => {
```

Comment:
Can you simply call `loadChat` without any parameter instead of creating a new method? we can call `loadChat()` to start a new conversation

Reply:
The `loadChat` only  work when current tab is chat tab. The loadChat will set current tab to CHAT in [this line](https://github.com/opensearch-project/dashboards-assistant/blob/main/public/hooks/use_chat_actions.tsx#L90). I think we can't change tab to CHAT if user is in history tab. So add a new `resetChat` here.